syncano/models/fields.py

```diff
@@ -12,6 +12,20 @@
 from .registry import registry
 
 
+class JSONToPythonMixin(object):
+
+    def to_python(self, value):
+        if value is None:
+            return
+
+        if isinstance(value, six.string_types):
```

Comment:
just wondering... because
```
if value is None:
    return
```
returns None and it's not string_type, then removing this condition will return None anyway...
but I'm not sure if explicit isn't better here